holoviews/plotting/bokeh/callbacks.py

```diff
@@ -10,7 +10,7 @@
 from ..comms import JupyterCommJS
 
 
-def attributes_js(attributes):
+def attributes_js(attributes, handles):
     """
```

Comment:
Would be good to see the docstring updated with an example showing how handles is involved...